lib/SPIRV/SPIRVToLLVMDbgTran.cpp

```diff
@@ -611,8 +620,19 @@ SPIRVToLLVMDbgTran::transTypeString(const SPIRVExtInst *DebugInst) {
                            0 /*AlignInBits*/, Encoding);
 }
 
-DINode *SPIRVToLLVMDbgTran::transTypeMember(const SPIRVExtInst *DebugInst) {
-  using namespace SPIRVDebug::Operand::TypeMember;
+DINode *
+SPIRVToLLVMDbgTran::transTypeMember(const SPIRVExtInst *DebugInst,
+                                    const SPIRVExtInst *ParentInst = nullptr,
```

Comment:
Maybe put default values in header file.

Reply:
Applied, thanks!